dls-controls/malcolmjs
# Feature/block title #137

## Description

Exposed widget for block label attribute and changed panel title to look at block.meta.label.
Also some under the hood changes to how deltas from block meta subscriptions are handled: deltas following the initial one are now applied, with only properties present in the payload being updated and new "blank" attributes being added only if they didn't already exist on the block.
 

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/gui/PANDA:INENC1 (or any block with an editable title (i.e. not PANDA))
- check the pane title matches the value in the "label" field
- edit the value of the label field and press enter to put the value to the server
- check that the title of the pane updates to match the new value

## Agile board tracking

connect to #137 
closes #137
